Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
I was leaving this as `net8.0` because `setup-dotnet` didn't yet have 9.x baked in and that added time to its setup time compared to .NET 8. It's a pretty small perf hit though, and it only affects the Train job since the Predictor app is cached.

So I could go either way.

Reply:
src/Directory.Build.props had it as net9.0, so I just kept it as is, just moved level up.
https://github.com/dotnet/issue-labeler/blob/9ff41c32db6f676853719d004d922f40dd5d2c9a/src/Directory.Build.props#L4

The whole set of apps are net9.0:
https://github.com/dotnet/extensions/actions/runs/13713580959/job/38354443416#step:4:13

We can downgrade to net8.0, if this is what you desired, though I can't see any compelling reason.